components/compositing/largest_contentful_paint_calculator.rs

```diff
@@ -60,7 +59,7 @@ impl LargestContentfulPaintCalculator {
         self.disabled_lcp_for_webviews.insert(webview_id);
     }
 
-    pub(crate) fn note_webview_removed(&mut self, webview_id: WebViewId) {
+    pub(crate) fn flush_webview_from_disable_list_of_webviews(&mut self, webview_id: WebViewId) {
```

Comment:
Let's just call this `enable_for_webview`. There's no need to expose the details of how things are enabled and disabled in the method name.

```suggestion
    pub(crate) fn enable_for_webview(&mut self, webview_id: WebViewId) {
```

Reply:
Done.